hw/ip/alert_handler/dv/cov/alert_handler_cov_excl.el

```diff
@@ -0,0 +1,21 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: chencindy
+// Format Version: 2
+// Date: Fri Oct  9 15:24:56 2020
+// ExclMode: default
+//==================================================
+CHECKSUM: "1270015083 3167692650"
+INSTANCE: tb.dut
+ANNOTATION: "[UNR] To reduce the simulation time, the max escalation cycle length is set to 1000."
+Toggle crashdump_o.class_esc_cnt [0][31:10] "logic [3:0][31:0]crashdump_o.class_esc_cnt"
+ANNOTATION: "[UNR] To reduce the simulation time, the max escalation cycle length is set to 1000."
+Toggle crashdump_o.class_esc_cnt [3][31:10] "logic [3:0][31:0]crashdump_o.class_esc_cnt"
+ANNOTATION: "[UNR] To reduce the simulation time, the max escalation cycle length is set to 1000."
+Toggle crashdump_o.class_esc_cnt [2][31:10] "logic [3:0][31:0]crashdump_o.class_esc_cnt"
+ANNOTATION: "[UNR] To reduce the simulation time, the max escalation cycle length is set to 1000."
+Toggle crashdump_o.class_esc_cnt [1][31:10] "logic [3:0][31:0]crashdump_o.class_esc_cnt"
```

Comment:
Hey @msfschaffner Do you have a sec to review this? This is the `esc_cnt` crashdump output that I excluded. As we mentioned in one of the meetings, in simulation, I constraint the esc phase_length from 2-1000 in order to reduce the simulation run time. So any esc_cnt above 1000 is unreachable.
We will review this exclusion again during the alert_handler signoff meeting.

Reply:
I feel this is more like LOW_RISK rather than UNR. Refer to https://docs.opentitan.org/doc/ug/dv_methodology/

> LOW_RISK: Items that are prohibitively hard to hit, given the resource constraints and are deemed to be of low risk and low value.

Reply:
That is true, I was using `UNR` because it is a constraint added in tb, and from the def: 
>UNR: Unreachable code due to constraints, or module inputs being tied off in a certain way will result in specific coverage items being unreachable.

But I can see your point, due to resource limit, it is hard to hit. I will change it to LOW_RISK. Thanks again for reviewing.

Reply:
By 'constraints' I was strictly referring to things in the RTL (example signals tied to particular values etc). I will update it. 

UNR items are things that are truly impossible to hit. This should be LOW_RISK since you are intentionally choosing to not even attempt to cover it.

Reply:
Awesome, thanks for clarifying, and thanks for catching that Weicai! I will update it.